### DISCUSSION TITLE
new instagram login

### DISCUSSION THREAD

**Discussion by no-1ne** [chatwoot/chatwoot#10113] [Feature Requests]
### Is your feature or enhancement related to a problem? Please describe.

https://developers.facebook.com/blog/post/2024/07/30/instagram-api-with-instagram-login/

### Describe the solution you'd like

simplified onboarding for end users

**Comment by linear** [comment]
<p><a href="https://linear.app/chatwoot/issue/PR-1332/new-instagram-login">PR-1332 new instagram login</a></p>

**Comment by angelfuel** [comment]
This would be a huge enhancement to chatwoot

**Comment by sajir0** [comment]
its very important to me please do it

**Comment by sadathoseini** [comment]
This feature will be a grate improvement
We are impatiently waiting for that

**Comment by haririalii** [comment]
It can be an interesting feature and it would improve the user experience.

**Comment by UniHowCoder** [comment]
After trying to onboard IG for hours today without success, this is definitely very interesting for me, really hope you're able to add this

**Comment by pranavrajs** [comment]
People who already have a working integration should not have any issues until December 4, 2024. However, new installations might not have access to the necessary permissions.

The change regarding Instagram is new, please see the update from Meta [here](https://developers.facebook.com/blog/post/2024/09/04/update-on-instagram-basic-display-api/). Our team is currently assessing the required updates. This requires a significant change on our end, and we'll be shipping updates before the current permission is completely phased out.

**Reply by sadathoseini** [reply]
Sorry, is it possible to have this feature in v4 release ?
